clusterloader2/testing/load/config.yaml

```diff
@@ -15,28 +18,63 @@
 {{$bigRCsPerNamespace := DivideInt $bigRCs $namespaces}}
```

Comment:
Same as in density test - let's assume that number of nodes is divisible by nodes per-namespace.

Reply:
Done. Test simplified.